test/plugs/movement_context_parser_test.exs

```diff
@@ -74,7 +74,7 @@ defmodule AccentTest.Plugs.MovementContextParser do
       |> assign(:project, project)
       |> MovementContextParser.call([])
 
-    assert conn.assigns[:document_parser] == (&Langue.Formatter.Json.Parser.parse/1)
+    assert conn.assigns[:document_parser] == &Langue.Formatter.Json.Parser.parse/1
```

Comment:
Same as your other PR: `mix format` should use Elixir 1.6.4 locally to pass on Travis (which run 1.6.4)